integ/src/monitor.rs

```diff
@@ -90,23 +90,30 @@ impl<T: BrupopClient> BrupopMonitor<T> {
 
     // verify if Burpop pods (agent, api-server, controller) are in `running` status.
     fn check_pods_health(&self, pods: &ObjectList<Pod>) -> bool {
```

Comment:
This function is pretty well self-contained. It would be a good idea to write unit tests for this function.

I would have thought that the `all()` call would cover this case!